03_python_fastapi_project/src/crud.py

```
@@ -1,8 +1,69 @@
 
 from sqlalchemy.future import select
 from sqlalchemy.ext.asyncio import AsyncSession
-from models import Product
+
+from models import Product, Cart, CartItem
 from schemas import ProductDTO, ProductCreate
+from sqlalchemy import select, and_
```

Ruff (0.12.2)

Redefinition of unused `select` from line 2

(F811)